assets/js/base/components/checkout/form-step/index.js

```diff
@@ -31,7 +31,9 @@ const StepNumber = ( { stepNumber } ) => {
 
 const StepHeading = ( { title, stepHeadingContent } ) => (
 	<div className="wc-block-checkout-step__heading">
-		<h4 className="wc-block-checkout-step__title">{ title }</h4>
+		<h4 className="wc-block-checkout-step__title">
+			<span aria-hidden="true">{ title }</span>
```

Comment:
Could the `aria-hidden` be moved to the `<h4>` tag, instead?

Reply:
ya and with that the span is unnecessary.